collections/_article/gdc-2025-retrospective.md

```diff
@@ -0,0 +1,60 @@
+
+## Public talk
+
+The Executive Director of the [Godot Foundation](https://godot.foundation), Emilio Coppola, gave [a talk](https://schedule.gdconf.com/session/building-godot-the-process-behind-the-engine-presented-by-w4-games/911990) at the GDC Partner Stage titled "Building Godot: The Process Behind the Engine".
+
+<img src="/storage/blog/gdc-2025/godot-talk.webp" alt="Emilio Coppola, executive director of the Godot Foundation, is at a podium giving his talk at the GDC." />
+
+Emilio explained in detail how the project works internally, and how the Foundation actively work for the benefit of everyone involved.
+
+The seats were almost full, but you'll have to trust me on this as I forgot to take a picture of the attendance. Though, I'm able to corroborate my claim with how many people were in line to have the chance to talk with Emilio. See it for yourself.
+
+<img src="/storage/blog/gdc-2025/godot-talk-after.webp" alt="Emilio Coppola answering questions from multiple talk attendees." />
+
+All of this wouldn't have been possible without the sponsor of the talk, [W4 Games](https://www.w4games.com/). Our many thanks to them.
+
+## Godot meetup
+
+Inspired by the successful [Blender meetup](https://bsky.app/profile/blender.org/post/3lk7d6lbhf22o) at the beginning of the week, we decided to try one on our own. [With only 2 days of notice](https://bsky.app/profile/adamscott.studio/post/3lkopw475z22y), we weren't expecting much.
+
+<img src="/storage/blog/gdc-2025/godot-meetup.webp" alt="Group picture of the Godot meetup participants at the Yerba Buena Gardens in San Francisco during the GDC" />
+
+But the Godot community shared the word and answered! We were stunned by the number of people that showed up.
+
+Those who attended the meetup had the opportunity to talk with regular Godot contributors, Emilio Coppola, and Ariel Manzur (co-founder of the Godot Engine and board member of the [Godot Foundation](https://godot.foundation)).
+
+Our many thanks to all of those who showed up! It was a pleasure to meet all of you. Fun fact: even if I called the end of the meetup at the 2-hour mark, somebody told me that the last people left nearly one hour after.
+
+## Showcased Godot games
+### Kneedle Drop: An Into the Woolderness Game
+
+"A RHYTHM KNITTING GAME?!" That's how [Immortal Funk Games](https://www.immortalfunkgames.com/) describes their spin-off game taylor-made (and chosen!) for the [alt.ctrl.GDC showcase of alternative controllers](https://gdconf.com/alt-ctrl-gdc).
+
+<img src="/storage/blog/gdc-2025/kneedle-drop-gameplay.webp" alt="Kneedle Drop gameplay, where two people hold each two controllers shaped as knitting needles." />
+
+In this game that mimic the essence of knitting as well as Guitar Hero does for playing guitar, either you tap the point of the needles, or you need to tap the left controller on the right one from one side or the other, all while keeping the rhythm.
```

Comment:
```suggestion
In this game, that mimics the essence of knitting as well as Guitar Hero does for playing guitar, either you tap the point of the needles, or you tap the left controller on the right one from one side or the other, all while keeping the rhythm.
```

Reply:
I'm not sure about the first comma, though.

Reply:
Yeah, it's fine without the first comma :)

Reply:
> Yeah, it's fine without the first comma :). I still think it's better (less clunky) without the "need to" though.

Oh, completely didn't see the "need to" suggestion. I'll remove it ASAP!